cmd/create.go

```diff
@@ -40,7 +44,8 @@ type createCmd struct {
 
 	supportedLangs *languages.Languages
 
-	templateWriter templatewriter.TemplateWriter
+	templateWriter   templatewriter.TemplateWriter
```

Comment:
💡 Nit: Extra spaces please.

Reply:
i will rerun gofmt on this file, but i think those spaces are to keep it aligned with the variable below